### ISSUE TITLE
Is this cookbook supported?

### ISSUE THREAD

**Issue by scopenco** [atomic-penguin/cookbook-gitlab-deprecated#127]
Hello,

Is this cookbook still supported by somebody?

**Comment by chewi** [comment]
I'm afraid I've left the company where I was maintaining it and we were also considering switching to an Omnibus installation.

**Comment by scopenco** [comment]
Do you have access to the supermaket gitlab account?

**Comment by chewi** [comment]
No, the gitlab name in the supermarket was handed over to GitLab for their official Omnibus-based cookbook. This cookbook takes a different approach and has never been official.
